src/routes/posts/posts.router.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+
+const {
+  createPostValidator,
+  updatePostValidator
+} = require('../../utils/validators');
+const { validatePost } = require('../../middleware/validate-post');
+
+const {
+  getAllPosts,
+  getPostById,
+  createPost,
+  updatePostById,
+  deletePostById
+} = require('./posts.controller');
+
+const router = express.Router();
+
+router.get('', getAllPosts);
+router.get('/:id', getPostById);
+router.put('', updatePostValidator, validatePost, updatePostById);
+router.post('', createPostValidator, validatePost, createPost);
+router.delete('', deletePostById);
```

Comment:
Sorry, straggler comment as I just noticed this - your routes for `put` and `delete` seem to be missing params. Ping me on slack if you don't know what I'm talking about!

Reply:
Good catch! I took the param from the request body - mainly because the project specs define the posts endpoints  like this https://www.notion.so/Project-Specs-project-and-feedback-groups-4d109441ca1a4e44a7db0a9003d02a6f I was about to confirm with the group to ensure whether we should use path param or follow the specs.

Reply:
I think it makes more sense to take the param from the path - so updated the route :)